csrankings.csv

```diff
@@ -13611,3 +13611,6 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Özgür Simsek,University of Bath,https://www.mpib-berlin.mpg.de/en/staff/oezguer-simsek,z1BYZG0AAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Andrew Lukefahr,Indiana University,http://homes.sice.indiana.edu/lukefahr/,tZ4fdX0AAAAJ
+Andrew Lumsdaine,Indiana University,https://www.sice.indiana.edu/faculty-research/directory/profile.html?profile_id=246,rc_EJ2kAAAAJ

```

Comment:
Andrew Lumsdaine is currently on a non-sabbatical leave to a national lab, so he does not meet the inclusion criteria.

Reply:
done!